packages/UI/src/components/atoms/NewButton/NewButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { tv, type VariantProps } from 'tailwind-variants';
+
+const button = tv({
+  base: 'text-center text-black  ',
+  variants: {
+    color: {
+      primary: 'bg-white shadow text-black text-center uppercase',
+      secondary: 'bg-red-400 text-white',
+    },
+    size: {
+      sm: ' rounded-lg py-2 px-4  flex flex-row gap-2 items-center justify-center',
+      md: ' rounded-lg px-4 py-2  flex flex-row gap-2 items-center ',
+      lg: ' rounded-lg px-4 py-3  flex flex-row gap-2 items-center ',
+    },
+  },
+  compoundVariants: [
+    {
+      size: ['sm', 'md'],
+      class: 'px-3 py-1',
+    },
+  ],
+  defaultVariants: {
+    size: 'md',
+    color: 'primary',
+  },
+});
+
+interface ButtonProps extends VariantProps<typeof button> {
```

Comment:
let's add all the types related to a button rather than just the custom ones we made, 

`interface ButtonProps extends VariantProps<typeof button>, React.DetailedHTMLProps<
      React.ButtonHTMLAttributes<HTMLButtonElement>,
      HTMLButtonElement
    >`